core/test/bundles/execWithStaticBundle/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+# This test is only applicable when build mode is shared libraries. 
+# The test point usExecutableBundleTest in usCoreDriver is sufficeint for static library builds.
+if(US_BUILD_SHARED_LIBS)
+
+set(BUILD_SHARED_LIBS 0)
+usFunctionCreateTestBundleWithResources(TestStaticBundle
```

Comment:
Suggestion; I think using "TestStaticallyLinkedBundle" is a more descriptive name to use than "TestStaticBundle". It uses the nomenclature that we want users to use (i.e. statically linked bundle).